packages/discord-reporter/src/index.ts

```diff
@@ -117,10 +134,13 @@ class DiscordReporter implements Reporter {
     const githubRunUrl = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}/actions/runs/${process.env.GITHUB_RUN_ID}`;
 
     const payload: WebhookPayload = {
+      content: resultToStatus[result.status].content,
       embeds: [
         {
           title: resultToStatus[result.status].title,
-          description: 'Here are the test run results:',
+          description: process.env.GH_JOB_NAME
```

Comment:
wdyt to pass this env as  `enabled` param - 
```
  discordReport: [
    '@lidofinance/discord-reporter',
    {
      enabled: process.env.DISCORD_REPORT_ENABLED,
    },
  ],
```

Reply:
its possible